accumulo/src/main/java/org/vertexium/accumulo/AccumuloGraph.java

```diff
@@ -2203,6 +2295,90 @@ protected GraphMetadataStore getGraphMetadataStore() {
         return graphMetadataStore;
     }
 
+    private CloseableIterable<ExtendedDataRow> getExtendedDataRowsInRange(
+            Span trace,
+            List<org.apache.accumulo.core.data.Range> ranges,
+            Authorizations authorizations
+    ) {
+        final long timerStartTime = System.currentTimeMillis();
+
+        return new LookAheadIterable<Map.Entry<Key, Value>, ExtendedDataRow>() {
+            public ScannerBase scanner;
+
+            @Override
+            protected boolean isIncluded(Map.Entry<Key, Value> src, ExtendedDataRow dest) {
+                return dest != null;
+            }
+
+            @Override
+            protected ExtendedDataRow convert(Map.Entry<Key, Value> next) {
+                try {
+                    SortedMap<Key, Value> row = WholeRowIterator.decodeRow(next.getKey(), next.getValue());
+                    ExtendedDataRowId extendedDataRowId = KeyHelper.parseExtendedDataRowId(next.getKey().getRow());
+                    return new AccumuloExtendedDataRow(extendedDataRowId, row, vertexiumSerializer);
+                } catch (IOException e) {
+                    throw new VertexiumException("Could not decode row", e);
+                }
+            }
+
+            @Override
+            protected Iterator<Map.Entry<Key, Value>> createIterator() {
+                try {
+                    scanner = createExtendedDataRowScanner(ranges, authorizations);
+                    return scanner.iterator();
+                } catch (RuntimeException ex) {
+                    if (ex.getCause() instanceof AccumuloSecurityException) {
+                        throw new SecurityVertexiumException("Could not get vertices with authorizations: " + authorizations, authorizations, ex.getCause());
+                    }
+                    throw ex;
+                }
+            }
+
+            @Override
+            public void close() {
+                super.close();
+                scanner.close();
+                if (trace != null) {
+                    trace.stop();
+                }
+                GRAPH_LOGGER.logEndIterator(System.currentTimeMillis() - timerStartTime);
+            }
+        };
+    }
+
+    private ScannerBase createExtendedDataRowScanner(List<org.apache.accumulo.core.data.Range> ranges, Authorizations authorizations) {
+        try {
+            String tableName = getExtendedDataTableName();
+            ScannerBase scanner;
+            if (ranges == null || ranges.size() == 1) {
+                org.apache.accumulo.core.data.Range range = ranges == null ? null : ranges.iterator().next();
+                scanner = createScanner(tableName, range, authorizations);
+            } else {
+                scanner = createBatchScanner(tableName, ranges, authorizations);
+            }
+
+            IteratorSetting versioningIteratorSettings = new IteratorSetting(
+                    90,
```

Comment:
Add variable name (or comment) to describe what the 90 (and 100 down below) values mean.